sources/packages/web/src/views/student/AppStudent.vue

```diff
@@ -151,14 +152,6 @@ export default defineComponent({
               },
             },
           },
-          {
-            title: "Overawards Balance",
-            props: {
-              to: {
-                name: StudentRoutesConst.STUDENT_OVERAWARDS,
-              },
-            },
-          },
```

Comment:
Instead of removing  the item and adding in case Full-Time is allowed I prefer leaving the list as it is and removing it in the `onMounted()` because that will be easier to maintain and to remove the condition one day.

```ts
    onMounted(async () => {
      const { isFulltimeAllowed } = await AppConfigService.shared.config();
      if (!isFulltimeAllowed) {
        menuItems.value.forEach((item, index) => {
          if (item.title === "Overawards Balance") {
            menuItems.value.splice(index, 1);
          }
        });
      }
    });
```

Reply:
updated